.circleci/config.yml

```diff
@@ -245,10 +260,12 @@ jobs:
       - run:
           name: Set up python by installing protobuf and enum
           command: pip install enum34==1.1.10 protobuf==3.13.0
+      - webpack_build:
+          to: ""
```

Comment:
'to' is a bit non-descriptive. Could we call it 'target_env' or similar, instead?

Also, we should probably explicitly specify "--prodenv" for prodenv tests since we also pass that flag to run_e2e_tests, and it's important to verify consistency. To that end, I suggest not having a default and require that all uses of the workflow specify the environment.

Reply:
Done!